test/e2e/azure/azure_helpers_test.go

```
@@ -0,0 +1,369 @@
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/msi/armmsi"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
+	"github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azsecrets"
+)
+
+// runCmd runs name with args, returning trimmed stdout. Both stdout and
+// stderr are captured; on failure, stderr is included in the returned error
+// for debuggability (matching how CI logs would otherwise show it). Used
+// only for the oc CLI -- all Azure resource CRUD below goes through
+// the Azure SDK for Go instead.
+func runCmd(name string, args ...string) (string, error) {
+	cmd := exec.Command(name, args...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)